lib/plug_datadog_stats.ex

```diff
@@ -48,6 +48,12 @@ defmodule PlugDatadogStats do
         "INT"
       String.match?(segment, ~r/^[a-f0-9]{8}(-[a-f0-9]{4}){3}-[a-f0-9]{12}$/)  ->
         "UUID"
+      String.match?(segment, ~r/p\w{6}$/) ->
+        "OBFUSCATED_ID"
+      String.match?(segment, ~r/q\w{13}$/) ->
+        "OBFUSCATED_ID"
+      String.match?(segment, ~r/r\w{25}$/) ->
```

Comment:
This should probably be start-of-string anchored with `^` at the beginning of the regex.